tests/test_attachments.py

```diff
@@ -140,11 +140,11 @@ async def default_handler(message: IncomingMessage, bot: Bot) -> None:
                 "pack": "4e4cfd0b-b981-54e9-84f4-3ddc12600334",
             },
         },
-        AttachmentSticker(
-            type=AttachmentTypes.STICKER,
+        Sticker(
             id=UUID("0dfd7318-2ccc-5384-b0e4-6fa5478606a5"),
             image_link="/uploads/sticker_pack/c81e87be5c7949b3b4196769e1032d5f.png",
             pack_id=UUID("4e4cfd0b-b981-54e9-84f4-3ddc12600334"),
+            emoji="/hello",
```

Comment:
Pls add `body` as first argument into `api_incoming_message_factory` and pass an emoji there.

Reply:
Done